### PR TITLE
Functional Tests : Theme Customization module - Uninstall and delete module

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Theme Customization module - Uninstall and delete module
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/7812060457
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by nesrineabdmouleh — DISMISSED** [review]

File: tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts [hunk 1]
```diff
@@ -90,6 +90,15 @@ export default class themeAndLogoBasePage extends BOBasePage {
     await this.waitForHiddenSelector(page, this.growlMessageBlock);
   }
 
+  /**
+   * Is present pages configuration tab
+   * @param page {Page} Browser tab
+   * @return {Promise<void>}
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts]:
```suggestion
   * @return {Promise<boolean>}
```

File: tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts [hunk 2]
```diff
@@ -99,6 +108,15 @@ export default class themeAndLogoBasePage extends BOBasePage {
     await this.clickAndWaitForURL(page, this.pagesConfigurationNavItemLink);
   }
 
+  /**
+   * Is present advanced customization tab
+   * @param page {Page} Browser tab
+   * @return {Promise<void>}
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts]:
```suggestion
   * @return {Promise<boolean>}
```

File: tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts]: Fixed

File: tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts (hunk 2 above)
Comment (Progi1984) [reply tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/themeAndLogoBasePage.ts]: Fixed

**Comment by Progi1984** [comment]
Ping @nesrineabdmouleh & @SD1982  for review

**Review by SD1982 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Pull request merged**